src/components/shared/FallbackImage.js

```diff
@@ -83,6 +83,10 @@ const FallbackImage = props => {
   }
 
   if (knownFallbackImages.includes(imageSource)) {
```

Comment:
fluid is semantic prop so for not semantic components its not need ( I was remove 'circle' so)

Reply:
I'm fixing the following error:
![image](https://user-images.githubusercontent.com/1877419/180349029-79d47f7d-17f7-42a0-8ebf-1639e505003f.png)

Reply:
Removed this code from this PR.